src/sentry/api/bases/organization.py

```diff
@@ -91,6 +103,116 @@ class OrganizationDiscoverSavedQueryPermission(OrganizationPermission):
 
 class OrganizationEndpoint(Endpoint):
     permission_classes = (OrganizationPermission, )
+    # Defines what happens if no date filter parameters are passed. If False,
+    # no date filtering occurs. If True, we provide default values.
+    DATE_FILTER_OPTIONAL = False
```

Comment:
Not totally sure this should be on the class, maybe could just parameterize `get_filter_params` more.

Reply:
yeah that might make it slightly less confusing, but either way

Reply:
Yeah, I'll move this to params